Tasks/UniversalPackagesV0/universalpublish.ts

```diff
@@ -153,6 +155,9 @@ function publishPackageUsingArtifactTool(publishDir: string, options: artifactTo
     const execResult: IExecSyncResult = artifactToolRunner.runArtifactTool(options.artifactToolPath, command, execOptions);
 
     if (execResult.code === 0) {
+        if(options.publishedPackageVar) {
+            tl.setVariable(options.publishedPackageVar, `${options.packageName}.${options.packageVersion}`);
```

Comment:
This could be hard to parse back out - '.' is allowed in both name and version isn't it?  How about a space as i can't think of a proto where that is a valid character of the name or version?  @mitchdenny ?

Reply:
I figured returning the exact name would probably be the correct thing but we could have a comma instead it would be much easier to parse.  There is a fancy regex you can use instead if we stick with the period: Regex.Split(PackageName, @"\.(?=\d)");

Reply:
Added space